packages/cli/src/commands/authorizations/create.ts

```diff
@@ -35,6 +35,12 @@ export default class AuthorizationsCreate extends Command {
 
     ux.action.stop()
 
+    const apiWarnings = headers['warning-message'] as string || ''
```

Comment:
Feels like maybe we could have a `Headers` type here with a `warning-message` field, but I think it's a minor thing. Maybe could think about that with the more robust solution to printing our warning messages everywhere.

Reply:
Yeah, I think we should create the `Headers` type when implementing the more robust solution. Thank you for pointing that out!